### PR TITLE
Min241006

### PR DESCRIPTION
**Pull request by minvp**
## Summary by CodeRabbit

- **New Features**
	- Introduced a function to replace email addresses in text with a placeholder.
	- Added a test for polling functionality to verify asynchronous logic on a webpage.

- **Tests**
	- Implemented a new test case for checking HTTP response status using polling.

### CODE REVIEW

File: daily-challenges/2024-09/21/min240921.spect.ts
Comment (minhphong306) [inline daily-challenges/2024-09/21/min240921.spect.ts:1]: Sửa lại tên file nha em. Spec thôi, ko phải spect

**Review by minhphong306 — CHANGES_REQUESTED** [review]
Please fix the comments.
Also see bot's comments.

File: daily-challenges/2024-09/21/min240921.spect.ts
Comment (minvp) [reply daily-challenges/2024-09/21/min240921.spect.ts:1]: vâng ạ. anh check giúp em nhé

**Review by minhphong306 — APPROVED** [review]
LGTM

**Pull request merged**